### ISSUE TITLE
Retiring support for old go package managers

### ISSUE THREAD

**Issue by julia-pu** [pivotal/LicenseFinder#845]
TLDR: We are looking for feedback on whether anyone is still using Glide/Dep/Trash

While attempting to upgrade to Go1.16, we are experiencing some issues with installing the following packages:
- Glide
- Dep
- Trash

Since it is recommended to use Go Modules now instead of these package managers, which have been either been deprecated or inactive for a long time, we would like to collect some feedback on whether there are people still using these. If not we would like to drop support for these package managers.

Notes:
[Glide](https://github.com/Masterminds/glide) hasn't had any changes since two years ago. In their [README](https://github.com/Masterminds/glide#go-modules) it says that Glide is mostly unmaintained and recommends the use of Go Modules instead.
[Dep](https://github.com/golang/dep) is deprecated and archived, and recommends the use of Go Modules in the [README](https://github.com/golang/dep#dep)
[Trash](https://github.com/rancher/trash) hasn't had any changes since two years ago.
